### ISSUE TITLE
peer list issue

### ISSUE THREAD

**Issue by mesouug** [wolandmaster/luci-app-rtorrent#8]
Good day.

Thanks for good product. I like it. Very simple installation and configuration.
I'm using raspberry pi openwrt version. Default generic installation and configuration.
All works good with exception of peers list.
When I try open peer list tab I'm getting following error:

```
/usr/lib/lua/luci/dispatcher.lua:433: Failed to execute cbi dispatcher target for entry '/admin/rtorrent/peers/537345734BF5189C1F62BD9634ECC54C85521C8A'.
The called action terminated with an exception:
...sr/lib/lua/luci/model/cbi/rtorrent/torrent/peers.lua:51: attempt to call a nil value
stack traceback:
	[C]: in function 'assert'
	/usr/lib/lua/luci/dispatcher.lua:433: in function 'dispatch'
	/usr/lib/lua/luci/dispatcher.lua:168: in function </usr/lib/lua/luci/dispatcher.lua:167>
```

Not sure what is missing.

Thanks,
Mikhail.

**Comment by Jcfunk** [comment]
Same with me,  on my router
```
Model   Buffalo WZR-HP-AG300H/WZR-600DHP
Firmware Version    OpenWrt Chaos Calmer 15.05 / LuCI (git-15.248.30277-3836b45)
Kernel Version  3.18.20
```
Here is error
```
/usr/lib/lua/luci/dispatcher.lua:433: Failed to execute cbi dispatcher target for entry '/admin/rtorrent/peers/337DDC5AF62CA2AE5C5B3FD2F193493D38B326E6'.
The called action terminated with an exception:
...sr/lib/lua/luci/model/cbi/rtorrent/torrent/peers.lua:51: attempt to call a nil value
stack traceback:
	[C]: in function 'assert'
	/usr/lib/lua/luci/dispatcher.lua:433: in function 'dispatch'
	/usr/lib/lua/luci/dispatcher.lua:168: in function </usr/lib/lua/luci/dispatcher.lua:167>
```

**Comment by wolandmaster** [comment]
Sorry for the late reply.
The previously used geoip site (www.telize.com) is not free anymore, this is the reason of peer list page failure. The problem fixed in the new version (0.1.4)
